test/bats/test.bats

```diff
@@ -16,7 +16,7 @@ SLEEP_TIME=1
 }
 
 @test "mutating webhook has been deployed" {
-    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io arc-osm-webhook-osm"
+    run wait_for_process $WAIT_TIME $SLEEP_TIME "kubectl get mutatingwebhookconfigurations.admissionregistration.k8s.io osm-webhook-osm"
```

Comment:
please open an issue in this repo to update this after the dependent chart has been updated to 0.5.1